docs/resources/README.md

```diff
@@ -0,0 +1,109 @@
+# Resources
+
```

Comment:
it would be good to give a background on what is resource etc. here in a follow up.